src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -16,6 +16,34 @@
 short-summary: Manage web apps.
 """
 
+helps['webapp auth'] = """
+    type: group
+    short-summary: Manage webapp authentication and authorization
+"""
+
+helps['webapp auth show'] = """
+    type: command
+    short-summary: Show the authentification settings for the webapp.
+"""
+
+helps['webapp auth update'] = """
+    type: command
+    short-summary: Update the authentication settings for the webapp.
+    examples:
+    - name: Enable AAD by enabling authentication and setting AAD-associated parameters. Default provider is set to AAD. Must have created a AAD service principal beforehand.
+      text: >
+        az webapp auth update  -g groupname -n webappname --enabled true \\
+          --action LoginWithAzureActiveDirectory \\
+          --aad-allowed-token-audiences <site-url>/.auth/login/aad/callback \\
+          --aad-client-id <aad-appId> --aad-client-secret <aad-password> \\
+          --aad-token-issuer-url <issuer; see param help for instructions>
```

Comment:
Not sure about this `<issuer; see param help for instructions>`.
@tjprescott / @sptramer thoughts?

Reply:
I was referring to the help output for the parameter when using -h, I can change the message to that?